test/roles.js

```diff
@@ -289,6 +289,26 @@ module.exports = function(app, template, hook) {
             done();
           });
       });
+
+      it('Should return error if undefined role id provided', done => {
+        request(app)
+          .get(hook.alter('url', '/role/undefined'))
```

Comment:
this test fails when run versus formio-enterprise (the hook tries to read from `undefined`)